### PR TITLE
5134: STEM-1995: Update Form Picker Wizard

### PR DESCRIPTION
**Pull request by elder-joseph**
## Description
As a veteran, I need to have a clear understanding of the eligibility requirements for the STEM Scholarship so that I do not complete and submit an application unnecessarily.

https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/5134

## Testing done
Testing passes locally.

## Screenshots
image
image

## Acceptance criteria
- [x] The "Updating my current education benefits" answer is replaced with "Updating my program of study or place of training"
- [x] The "Applying to extend my benefit using the Edith Nourse Rogers STEM Scholarship" answer is replaced with "Applying to extend my Post-9/11 or Fry Scholarship benefits using the Edith Nourse Rogers STEM Scholarship"
- [x] If the user selects "Applying to extend my Post-9/11 or Fry Scholarship benefits using the Edith Nourse Rogers STEM Scholarship" is selected, the following content is displayed:
To be eligible for the Edith Nourse Rogers STEM Scholarship, you must meet all the requirements below. You:
• Are receiving Post-9/11 GI Bill or Fry Scholarship benefits
• Have used up all your education benefits or are within 6 months of using all your benefits. Check remaining benefits
• Are enrolled in an undergraduate program for science, technology, engineering or math (STEM), or have already earned a STEM degree and are pursuing a teaching certification. See approved STEM programs

To learn more about the scholarship, visit the VBA STEM page.

- [x] If the user selects "Applying to extend my Post-9/11 or Fry Scholarship benefits using the Edith Nourse Rogers STEM Scholarship" is selected, the following question is displayed after the text in AC # 3: "Based on the eligibility requirements above, do you want to apply for this scholarship?"
- [x] The following answers are available for the answer to the question in AC # 4:
a. Yes
b. No
- [x]If the user selects "Yes" as the answer to question in AC # 4, the "Apply Now" button is displayed.
- [x] If the user selects "No" as the answer to question in AC # 4, the following content is displayed: "Learn what other education benefits you may be eligible for on the GI Bill eligibility page."
GI Bill eligibility page is a link to the following url: https://www.va.gov/education/eligibility/

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/edu-benefits/components/EducationWizard.jsx [hunk 1]
```diff
@@ -281,6 +299,73 @@ export default class EducationWizard extends React.Component {
...
+                        for on the{' '}
+                        <a href="https://www.va.gov/education/eligibility/">
+                          GI Bill eligibility page.
+                        </a>
+                      </p>
+                    ))}
+                </div>
+              </div>
+            ) : (
+              newBenefit === 'extend' && this.getButton('1995')
```
Comment (shawkey-daniel) [inline src/applications/edu-benefits/components/EducationWizard.jsx]: newBenefit === 'extend' is already established before this ternary.

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by delli-gatti-michael — APPROVED** [review]

**Review by bkjohnson — COMMENTED** [review]
It looks like there are some linting errors here.

**Comment by elder-joseph** [comment]
Hey @bkjohnson

Sorry about that, looks like something got out of sync locally for me. Fixed that, and my latest commit seems to have fixed the linting issues for this PR

**Review by bkjohnson — APPROVED** [review]
:+1:

**Pull request merged**